### ISSUE TITLE
Export graph -> no rendering of HTML entities

### ISSUE THREAD

**Issue by AnaelMobilia** [highcharts/highcharts#15411]
#### Expected behaviour
HTML entities are rendered on the graph due to the use of ```labels -> useHTML: true,```
On the export of the graph (PNG images for example), HTML entities are translated

#### Actual behaviour
On the export of the graph (PNG images for example), HTML entities are displayed as is

#### Live demo with steps to reproduce
http://jsfiddle.net/u35hm8de/1/

A workaround can be used through ```Highcharts.SVGRenderer.prototype.escapes['→'] = '&rarr;'```

#### Product version
Highcharts 9.0.1

#### Affected browser(s)
Firefox 86

**Comment by KacperMadej** [comment]
Hi @AnaelMobilia 

If you will set `exporting.allowHTML:true` then it works fine.
API reference: https://api.highcharts.com/highcharts/exporting.allowHTML
Demo: http://jsfiddle.net/BlackLabel/n86197vp/

**Comment by AnaelMobilia** [comment]
Hi @KacperMadej 
This option require to use the Highcharts export server and can't be run inside the browser.

Is it a way to render it locally ? Maybe define the ```Highcharts.SVGRenderer.prototype.escapes``` for all usual HTML entities ?

Bests regards

**Comment by KacperMadej** [comment]
> This option require to use the Highcharts export server and can't be run inside the browser.

How could this be a problem in this issue? This issue is about exporting and your demo shows export server usage.

For client-side export see https://www.highcharts.com/docs/export-module/client-side-export

**Comment by AnaelMobilia** [comment]
Indeed, this jsfiddle include an updated example includind the offline-export => http://jsfiddle.net/0qvyoej3/ 

For the use case : sometimes customers use sensitives datas which can't be transmitted to a third party server ;-)
